wire-ios-sync-engine/Source/UserSession/UserSession.swift

```diff
@@ -91,6 +146,10 @@ public protocol UserSession: AnyObject {
         for: UserType
     ) -> NSObjectProtocol?
 
+    func addUserObservers(
```

Comment:
```suggestion
    func addUserObserver(
```

We are only one observer

Reply:
Done!